index.js

```diff
@@ -25,17 +24,26 @@ class Filru {
         throw new TypeError("filru: maxAge must be a number greater than 0");
       }
     }
+    if (typeof hashSeed !== "number" && typeof hashSeed !== "string") {
+      throw new TypeError("filru: hashSeed must be a number or a string");
+    }
+    if (typeof pruneInterval !== "number" || pruneInterval < 1) {
+      throw new TypeError("filru: pruneInterval must be a number greater than 0");
+    }
+
     this.dir = dir;
     this.maxBytes = maxBytes;
     this.maxAge = maxAge;
+    this.hashSeed = hashSeed;
+    this.pruneInterval = pruneInterval;
     this.stopped = false;
     this.load = loadFunc;
 
     this._timeout = null;
   }
 
-  static hash(key) {
```

Comment:
This is a breaking API change, taking it from:
`Filru.hash('filename')` (class method)
to
`new Filru().hash('filename')` (instance method)

It may require a little more thought. I recommend keeping original behavior, and adding a new instance method.